### PR TITLE
Update references to AdoptOpenJDK to Adoptium

### LINKED ISSUE #2558: Update references to AdoptOpenJDK to Adoptium
There are some references to Github organizations and repos that need updating in this repository.  Now that all test materials are moved to Adoptium, we should search for and replace the references to any that are no longer valid (even though Github forwarding will mostly handle it, we should update nonetheless).  Org names and repo names need to be searched for and updated.

Test repos moved/renamed:
adoptopenjdk/openjdk-tests -> adoptium/aqa-tests
adoptopenjdk/openjdk-systemtest -> adoptium/aqa-systemtest
adoptopenjdk/openjdk-test-tools -> adoptium/aqa-test-tools
adoptopenjdk/STF -> adoptium/STF
adoptopenjdk/TKG -> adoptium/TKG
adoptopenjdk/bumblebench -> adoptium/bumblebench
adoptopenjdk/run-aqa -> adoptium/run-aqa

Other potentially referenced repos moved/renamed:
adoptopenjdk/build-jdk -> adoptium/build-jdk

### PR DESCRIPTION
**Pull request by 2001asjad**
Update references to AdoptOpenJDK to Adoptium in various places in this repository.

Fixes #2558

### CODE REVIEW

**Comment by 2001asjad** [comment]
@smlambert @llxia I have made a PR. Can you take a look so that we can discuss more about it.
PS: There were some conflicts which I have merged.
Thank You

**Comment by llxia** [comment]
This PR contains duplicate changes of https://github.com/adoptium/aqa-tests/pull/2542/ FYI @Mesbah-Alam

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -41,8 +41,10 @@ def setupEnv() {
 
 	env.SPEC = "${SPEC}"
 
-	ADOPTOPENJDK_REPO = params.ADOPTOPENJDK_REPO ? params.ADOPTOPENJDK_REPO : "https://github.com/AdoptOpenJDK/openjdk-tests.git"
-	OPENJ9_REPO = params.OPENJ9_REPO ? params.OPENJ9_REPO : "https://github.com/eclipse-openj9/openj9.git"
+
+	ADOPTOPENJDK_REPO = params.ADOPTOPENJDK_REPO ? params.ADOPTOPENJDK_REPO : "https://github.com/adoptium/aqa-tests.git"
+	OPENJ9_REPO = params.OPENJ9_REPO ? params.OPENJ9_REPO : "https://github.com/eclipse/openj9.git"
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase:46]: The original `https://github.com/eclipse-openj9/openj9.git` is correct. Please do not change the original value.

**Comment by llxia** [comment]
I would suggest removing the change-related aqa-systemtest as it is been taken care of in https://github.com/adoptium/aqa-tests/pull/2542

File: .github/workflows/runAqa.yml [hunk 2]
```diff
@@ -113,14 +113,14 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJson(needs.parseComment.outputs.build_parameters) }}
     steps:
-    - uses: AdoptOpenJDK/install-jdk@v1
+    - uses: adoptium/install-jdk@v1
```
Comment (llxia) [inline .github/workflows/runAqa.yml:116]: This is invalid

File: .github/workflows/autoTestPR.yml [hunk 3]
```diff
@@ -53,4 +53,4 @@ jobs:
           signoff: 'true'
       - name: add label
         run: |
-          curl -u github-actions:${{ secrets.GITHUB_TOKEN }} -d '{"labels":["test excluded"]}' -X POST ${{ github.event.issue.url }}/labels
\ No newline at end of file
+          curl -u github-actions:${{ secrets.GITHUB_TOKEN }} -d '{"labels":["test excluded"]}' -X POST ${{ github.event.issue.url }}/labelsadoptium/TKG
```
Comment (llxia) [inline .github/workflows/autoTestPR.yml:56]: Please remove extra adoptium/TKG

File: README.md [hunk 4]
```diff
@@ -11,61 +11,61 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+For nightly and release builds, there are test jobs running as part of the adoptium continuous delivery pipelines.  There is a [blog post and brief presentation](https://blog.adoptium.net/2017/12/testing-java-help-count-ways) that explains what testing we run and how they fit into the overall delivery pipeline.  As the world of testing at adoptium is evolving and improving quickly, some documentation may fall behind the march of progress.  Please let us know and help us keep it up-to-date, and ask questions at the [adoptium testing Slack channel](https://adoptium.slack.com/messages/C5219G28G)!
```
Comment (llxia) [inline README.md:18]: https://blog.adoptopenjdk.net/2017/12/testing-java-help-count-wayURL should not be changed

File: README.md [hunk 5]
```diff
@@ -11,61 +11,61 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
 
 ![CI pipeline view](doc/diagrams/ciPipeline.jpg)
 
 #### Test 'Inventory'
 
-The directory structure in this openjdk-tests repository is meant to reflect the different types of test we run (and pull from lots of other locations).  The diagrams below show the test make target for each of the types, along with in-plan, upcoming additions (denoted by dotted line grey boxes). The provided links jump to test jobs in Jenkins (ci.adoptopenjdk.net).
+The directory structure in this openjdk-tests repository is meant to reflect the different types of test we run (and pull from lots of other locations).  The diagrams below show the test make target for each of the types, along with in-plan, upcoming additions (denoted by dotted line grey boxes). The provided links jump to test jobs in Jenkins (ci.adoptium.net).
```
Comment (llxia) [inline README.md:24]: We donot have ci.adoptium.net

**Comment by llxia** [comment]
Please only update the repos mentioned in https://github.com/adoptium/aqa-tests/issues/2558#issue-874611199.

**Comment by 2001asjad** [comment]
@llxia can you check it now please.
Thank You

File: buildenv/jenkins/testJobTemplate [hunk 6]
```diff
@@ -124,8 +124,10 @@ println "JDK_VERSIONS: ${JDK_VERSIONS}"
 println "GROUPS: ${GROUPS}"
 println "ARCH_OS_LIST: ${ARCH_OS_LIST}"
 
-def OPENJ9_REPO = "https://github.com/eclipse-openj9/openj9.git"
-def ADOPTOPENJDK_REPO = "https://github.com/AdoptOpenJDK/openjdk-tests.git"
+
+def OPENJ9_REPO = "https://github.com/eclipse/openj9.git"
```
Comment (llxia) [inline buildenv/jenkins/testJobTemplate:128]: The original value `https://github.com/eclipse-openj9/openj9.git` is correct. Please do not change it.

**Comment by llxia** [comment]
Thanks @2001asjad  
The above comments are not fully addressed. For example, there are still lots of` adoptium/install-jdk` . We should not change `adoptopenjdk` to `AdoptOpenJDK`. Please also address https://github.com/adoptium/aqa-tests/pull/2567#issuecomment-832918936

**Comment by 2001asjad** [comment]
> Thanks @2001asjad
> The above comments are not fully addressed. For example, there are still lots of` adoptium/install-jdk` . We should not change `adoptopenjdk` to `AdoptOpenJDK`. Please also address [#2567 (comment)](https://github.com/adoptium/aqa-tests/pull/2567#issuecomment-832918936)

So you are saying I don't need to change the one's that are done in the draft of issue no [2542](https://github.com/adoptium/aqa-tests/pull/2542/files)?

**Comment by llxia** [comment]
Please resolve the conflicts. Thanks

**Comment by Mesbah-Alam** [comment]
Please also rebase your branch to accept the latest changes delivered via https://github.com/adoptium/aqa-tests/pull/2542 and refresh this PR. Heads up: this could introduce some conflicts that may need to be resolved manually.

**Comment by smlambert** [comment]
run aqa --help

**Comment by smlambert** [comment]
@2001asjad - based on our meeting last week, I think you mentioned you will close this PR and open a new one to replace it.  I am just adding a note here as a reminder.  Thanks!

**Comment by 2001asjad** [comment]
> @2001asjad - based on our meeting last week, I think you mentioned you will close this PR and open a new one to replace it. I am just adding a note here as a reminder. Thanks!

Yes @smlambert I remember it of course. 
Gonna work on it tonight.

**Pull request closed without merging**